src/Search_Replace_Command.php

```diff
@@ -547,7 +547,14 @@ private function write_sql_row_fields( $table, $rows ) {
 			if( ( $index % $export_insert_size == 0 && $index > 0 ) || $index == $count ) {
 				$sql .= ";\n";
 
-				$sql = $wpdb->prepare( $sql, array_values( $values ) );
+				if( method_exists( $wpdb, 'remove_placeholder_escape' ) ) {
+					// since 4.8.3
+					$sql = $wpdb->remove_placeholder_escape( $wpdb->prepare( $sql, array_values( $values ) ) );
```

Comment:
Is there a reason we can't internalize `remove_placeholder_escape` and use it with earlier WP versions?

Reply:
You don't need it for earlier WP versions. WP internal behavior changed in 4.8.3 and the method remove_placeholder_escape was provided at the same time to help plugin developers:
https://make.wordpress.org/core/2017/10/31/changed-behaviour-of-esc_sql-in-wordpress-4-8-3/